.github/workflows/ci_android_release.yaml

```diff
@@ -0,0 +1,82 @@
+#on: [push, pull_request]
+#
```

Comment:
why is this file commented out?

Reply:
It is duplicated because now that is on the main release one. But I wanted to keep as a base for android tests in the future maybe.